# 
# Submitted:
#
# Robert Schwebel, 2004-06-22
#
# Error: 
# 
# gcc-3.3.2 claims about a missing errno symbol. 
#
# Description: 
#
# State:
# 
# sitting here, waiting for upstream feed...
#

diff -urN portmap_4-orig/portmap.c portmap_4/portmap.c
--- portmap_4-orig/portmap.c	2004-06-22 21:19:12.000000000 +0200
+++ portmap_4/portmap.c	2004-06-22 21:23:21.000000000 +0200
@@ -85,6 +85,7 @@
 #include <stdio.h>
 #include <syslog.h>
 #include <netdb.h>
+#include <errno.h>
 #include <sys/socket.h>
 #include <sys/ioctl.h>
 #include <sys/wait.h>
